tests/platform_tests/api/test_watchdog.py

```diff
@@ -151,34 +174,13 @@ def test_arm_too_big_timeout(self, duthost, platform_api_conn, conf):
         if watchdog_timeout is None:
             pytest.skip('"too_big_timeout" parameter is required for this test case')
         actual_timeout = watchdog.arm(platform_api_conn, watchdog_timeout)
-
-        assert actual_timeout == -1
+        self.expect(actual_timeout == -1, "{}: watchdog time {} shouldn't be set".format(test_arm_too_big_timeout.__name__, watchdog_timeout))
+        self.assert_expectations()
 
     def test_arm_negative_timeout(self, duthost, platform_api_conn):
         ''' try to arm the watchdog with negative value '''
 
         watchdog_timeout = -1
         actual_timeout = watchdog.arm(platform_api_conn, watchdog_timeout)
-
-        assert actual_timeout == -1
-
-    @pytest.mark.disable_loganalyzer
-    def test_reboot(self, duthost, localhost, platform_api_conn, conf):
-        ''' arm the watchdog and verify it did its job after timeout expiration '''
-
-        watchdog_timeout = conf['valid_timeout']
-        actual_timeout = watchdog.arm(platform_api_conn, watchdog_timeout)
-
-        assert actual_timeout != -1
-
-        res = localhost.wait_for(host=duthost.hostname, port=22, state="stopped", delay=2,
-                                 timeout=actual_timeout + TIMEOUT_DEVIATION,
-                                 module_ignore_errors=True)
-        assert 'exception' not in res
-
-        res = localhost.wait_for(host=duthost.hostname, port=22, state="started", delay=10, timeout=120,
-                                 module_ignore_errors=True)
-        assert 'exception' not in res
-
-        # wait for system to startup
-        time.sleep(120)
+        self.expect(actual_timeout == -1, "{}: watchdog time {} shouldn't be set".format(test_arm_negative_timeout.__name__, watchdog_timeout))
```

Comment:
Suggest changing message to,

```
"{}: Watchdog should be disarmed, but returned timeout of {} seconds".format(test_arm_negative_timeout.__name__, watchdog_timeout))
```

Reply:
Please use my suggested wording here. It is clear and simple, and the test name will get prefixed to the error message, so we will know which test failed with this message.